content/learn/book/contributing/_index.md

```diff
@@ -3,10 +3,11 @@ title = "Contributing"
 weight = 4
 sort_by = "weight"
 template = "docs-section.html"
-page_template = "docs-section.html"
+page_template = "docs-page.html"
 insert_anchor_links = "right"
 [extra]
 subtitle = "join the bevy"
+weight = 5
```

Comment:
Why is duplicating the "weight" field necessary? Remembering to add it twice is extra work that needs to be done, and it creates potential for the weight to be out of sync.
This one is out of sync with the "actual weight". Was that a mistake or intentional?

Reply:
In the docs menu macro we can't access the actual weight thru the zola pages / sections objects. It isn't a field. But I needed to resort them by weight due to the fact there isn't a mixed pages and sections array and I had to make one myself.

The out of sync wasn't intended.

Tbh, I'm 99% sure we can just replace zola's weight with the custom weights and be fine. I just left them in on the off chance I missed one esoteric thing that does need weights and the code I added doesn't replace in effective functionality and as such breaks things unexpectedly.

Reply:
Okay, have removed the duplication.